doc/syslog/syslog-design.md

```diff
@@ -107,69 +128,97 @@ This might be useful for security reasons.
+6. Syslog filter configuration
 
-### 1.2.2 Command interface
+### 2.2.2. Command interface
 
 **This feature will support the following commands:**
 1. config: add/delete syslog server configuration
 2. show: display syslog server configuration
+3. config: set/unset global syslog configuration
+4. show: display global syslog configuration
 
-### 1.2.3 Error handling
+### 2.2.3. Error handling
 
 **This feature will provide error handling for the next situations:**
 1. Invalid object reference
 2. Invalid options/parameters
 
-### 1.2.4 Event logging
+### 2.2.4. Event logging
 
 **This feature will provide event logging for the next situations:**
 1. Syslog server add/delete
 
-###### Table 1: Event logging
+#### 2.2.4.1. Table 1: Event logging
 
 | Event                             | Severity |
 |:----------------------------------|:---------|
 | Syslog server add/delete: success | NOTICE   |
 | Syslog server add/delete: error   | ERROR    |
 
-# 2 Design
+# 3. Design
 
-## 2.1 Overview
+## 3.1. Overview
 
 SSIP will reuse syslog `omfwd` functionality which offers the next features:  
 1. Source IP address configuration
-2. Server UDP port configuration
+2. Server port configuration
 3. VRF device configuration
+4. Server protocol configuration
+5. Messages regex filtering 
+6. Messages severity filtering
 
-## 2.2 Syslog Forwarding Output Module
+## 3.2. Syslog Forwarding Output Module
 
-### 2.2.1 Overview
+### 3.2.1. Overview
 
-###### Table 2: Syslog Forwarding Output Module
+#### 3.2.1.1. Table 2: Syslog global config Module
 
-| Parameter  | Default  | Description                                                                               |
-|:-----------|:---------|:------------------------------------------------------------------------------------------|
-| target     | none     | Name or IP-Address of the system that shall receive messages. Any resolvable name is fine |
-| address    | none     | Bind socket to a given local IP address. This option is only supported for UDP, not TCP   |
-| port       | 514      | Name or numerical value of port to use when connecting to target                          |
-| protocol   | udp      | Type of protocol to use for forwarding (e.g., tcp/udp)                                    |
-| device     | none     | Bind socket to given device (e.g., eth0/vrf0)                                             |
-| ipfreebind | 2        | Manages the IP_FREEBIND option on the UDP socket                                          |
+| Parameter          | Default  | Description                                                                   |
+|:-------------------|:---------|:------------------------------------------------------------------------------|
+| format             | standard | template format                                                               |
+| trap               | notice   | messages with severity equal or grater then this severity will be forwarded   |
```

Comment:
severity will be a better name as trap may confuse with SNMP terminology. Please keep uniform name for both global & server specific configuration parameters.

Reply:
we defined this name in our internal usage so we would prefer to leave it like that.
is it acceptable by you?